server/src/app/index.js

```diff
@@ -0,0 +1,34 @@
+import {
+  express,
+  index,
+  users,
+  mongoose,
+  dotenv,
+  bodyParser
+} from "./constants/packageConfig";
+
+dotenv.config({ path: "../.env" });
+
+const port = process.env.PORT || 5000;
+const app = express();
+
+mongoose
+  .connect(process.env.MONGODB_URI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useCreateIndex: true
+  })
+  .then(() => console.log("Mongo DB Connected"))
+  .catch(err => console.log(err));
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.use("/", index);
+app.use("/users", users);
+
+app.listen(port, () => {
```

Comment:
Start the server after the database is connected.

Reply:
Done